pkg/util/cluster.go

```diff
@@ -58,12 +59,21 @@ type ClusterInfoImp struct {
 	monitoringAvailable           bool
 	deschedulerAvailable          bool
 	singlestackipv6               bool
+	masterNodesCount              int
```

Comment:
These two new fields can be a method variable. I don't think we need them in the type.

Reply:
this is not needed at all eventually.